src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -190,6 +190,13 @@ Json::Value SotaUptaneClient::AssembleManifest() {
   Json::Value version_manifest;
 
   Json::Value primary_ecu_version = package_manager_->getManifest(primary_ecu_serial);
+  std::vector<std::pair<Uptane::EcuSerial, int64_t>> ecu_cnt;
+  if (!storage->loadEcuReportCounter(&ecu_cnt) || (ecu_cnt.size() == 0)) {
+    LOG_ERROR << "No ECU version report counter, please check the database!";
+  } else {
+    primary_ecu_version["report_counter"] = std::to_string(ecu_cnt[0].second + 1);
```

Comment:
@simao @jochenschneider If we add the ECU version report counter here, does that look good to you? You can maybe see it better in the test below (https://github.com/advancedtelematic/aktualizr/pull/1451/files#diff-81426acc9fdba515977c8106aaa0eec4R159).

Reply:
Is that field documented somewhere? It is hard to see the context here.

Reply:
Is https://saeljira.it.here.com/browse/OTA-3864 enough context?

But actually, either way it makes sense to make this more obvious. @xcheng-here can you post an example of what a manifest with the counter would look like? You can get it from the logs while running manually or by debugging the test case that uses it.

Reply:
Yes, I'd love to have an example.

Reply:
Sure, I uploaded a manifest.txt to the jira case, it's at line 94.